copier.yaml

```diff
@@ -2,22 +2,28 @@ _envops:
   block_end_string: '%]'
   block_start_string: '[%'
 _subdirectory: template
+_templates_suffix: .jinja
+
```

Comment:
May I know what do you use to format this file? I think we should make it reproducible and maintained the corresponding config in the repo (maybe in a different pull request?)

Reply:
As far as now, I maitain it with blank lines mannually for more readability. But there is a yaml fix hook  https://github.com/macisamuele/language-formatters-pre-commit-hooks based on ruamel.

Reply:
@msclock I may prefer some tools that keep the format consistent and reproducible even if it is not that perfect. Anything maintained manually will break up in the end especially when involving more and more teammates. That is why I choose the yaml sort vscode plugin.